### PR TITLE
Fixup image buffer dump

### PR DESCRIPTION
**Pull request by dj2**
This CL updates the image dumping code to the new default framebuffer format of B8G8R8A8_UNORM.

### CODE REVIEW

**Comment by dj2** [comment]
@hevrard I believe this should fix the issue you encountered.

File: samples/ppm.cc [hunk 1]
```diff
@@ -52,10 +52,10 @@ std::pair<amber::Result, std::string> ConvertToPPM(
   // Write PPM data
   for (amber::Value value : values) {
     const uint32_t pixel = value.AsUint32();
-    // We assume R8G8B8A8_UINT here:
-    image.push_back(byte0(pixel));  // R
+    // We assume B8G8R8A8_UNORM here:
+    image.push_back(byte2(pixel));  // R
```
Comment (hevrard) [inline samples/ppm.cc]: The comment is out of sync here, it should read BGRA

File: samples/ppm.cc [hunk 2]
```diff
@@ -52,10 +52,10 @@ std::pair<amber::Result, std::string> ConvertToPPM(
...
     image.push_back(byte1(pixel));  // G
-    image.push_back(byte2(pixel));  // B
+    image.push_back(byte0(pixel));  // B
```
Comment (hevrard) [inline samples/ppm.cc]: idem comment out of sync

File: samples/ppm.cc (hunk 1 above)
Comment (dj2) [reply samples/ppm.cc]: Hah, good catch. Updated the top comment and missed all the inline ones.

Comment (dj2) [reply samples/ppm.cc]: Actually, I think these are right. We're writting the R value here, not the B value. It just happens that R is in byte2.

Does that make sense?

Comment (hevrard) [reply samples/ppm.cc]: Woops my bad, I got confused, the comments are correct indeed!

**Pull request merged**